cmd/minishift/cmd/start.go

```diff
@@ -57,6 +57,7 @@ const (
 
 var (
 	dockerEnv        []string
+	dockerEngineOpt  []string
```

Comment:
Still not quite sure where this is then picked up. Do we not need any changes to our provisioner now?